src/solver/optimisation/opt_appel_solveur_quadratique.cpp

```diff
@@ -21,24 +21,22 @@
 
 #include <limits>
 
-#include "antares/solver/optimisation/opt_fonctions.h"
-#include "antares/solver/optimisation/opt_structure_probleme_a_resoudre.h"
-
 /*
  pi_define.h doesn't include this header, yet it uses struct jmp_buf.
  It would be nice to remove this include, but would require to change pi_define.h,
  which isn't part of Antares
 */
-#include <setjmp.h>
 
+#include <setjmp.h>
 extern "C"
 {
-#include "pi_define.h"
-#include "pi_definition_arguments.h"
-#include "pi_fonctions.h"
+#include <pi_constantes_externes.h>
+#include <pi_definition_arguments.h>
+#include <pi_fonctions.h>
 }
```

Comment:
`#include <setjmp.h>` : 
We should make a ticket about this.
As the related comment says, this inclusion should belong to **Sirius**, but because **Sirius** does not do it, **Antares** has to do it instead, or code does not compile.
Why not change **Sirius** ? Any reason ?

Reply:
No particular reason, we don't want to touch Sirius too much (deprecated)